libs/application/templates/financial-statements-inao/src/shared/constants.ts

```diff
@@ -0,0 +1,5 @@
+export enum ApiActions {
+  createApplication = 'createApplication',
+  doStuffThatFails = 'doStuffThatFails',
+  completeApplication = 'completeApplication',
```

Comment:
Is this used anywhere? Should probably be removed if not.

Reply:
removed